docs/embeds.md

```diff
@@ -103,28 +103,39 @@ export function EmbedChart() {
 
 <div class="tip">
 
-Since both dynamic import and the imported component are async, the code above is careful to clean up the effect and avoid race conditions.
+Some web tooling such as Vite and Webpack erroneously rewrite external dynamic imports. You may need to include a comment such as `import(/* @vite-ignore */ …)` or `import(/* webpackIgnore: true */ …)` to disable this behavior.
 
 </div>
 
-<div class="tip">
+## Exported files
 
-You can alternatively embed Framework pages using [iframe embeds](https://observablehq.observablehq.cloud/framework-example-responsive-iframe/).
+In addition to modules, you can declare specific files to export using the [**dynamicPaths** config option](./config#dynamic-paths). Exported files are published under a stable URL that can be linked to and loaded from an external application. Exported files can be either [static files](./files) or generated dynamically by [data loaders](./data-loaders), and can use [parameterized routes](./params). For example, to export the file `/robots.txt`:
 
-</div>
+```js run=false
+export default {
+  dynamicPaths: [
+    "/robots.txt"
+  ]
+};
+```
 
-## Developing modules
+## Iframe embeds
 
-To develop your component, you can import it into a Framework page like normal, giving you instant reactivity as you make changes to the component or its data.
+You can alternatively embed Framework pages using iframes. Pages that are intended to be embedded via iframe typically disable Framework’s built-in user interface using [Markdown front matter](./markdown#front-matter):
 
-```js echo
-import {Chart} from "./chart.js";
+```yaml
+---
+sidebar: false
+header: false
+footer: false
+pager: false
+---
```

Comment:
I wonder if we should add a shorthand for this, so people could say “this page has no chrome” without having to list everything that should not be there?

Reply:
Ya, `chrome: false` sounds nice. 👍